S/Spot_julia/build_tarballs.jl

```diff
@@ -0,0 +1,99 @@
+    ]
+    
+    # Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/spot-2.9.7/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-python
+make -j${nproc}
+make install
+
+# build with cmake 
+cd $WORKSPACE/srcdir/spot_julia/spot_julia
+
+if [[ $target == *"mingw"* ]]; then
+  cp -L ${prefix}/bin/*.dll ${prefix}/lib/
+  rm ${prefix}/bin/*.dll
+fi
+
+# Override compiler ID to silence the horrible "No features found" cmake error
+if [[ $target == *"apple-darwin"* ]]; then
+  macos_extra_flags="-DCMAKE_CXX_COMPILER_ID=AppleClang -DCMAKE_CXX_COMPILER_VERSION=10.0.0 -DCMAKE_CXX_STANDARD_COMPUTED_DEFAULT=11"
+fi
+Julia_PREFIX=$prefix
+mkdir build
+cd build
+cmake -DJulia_PREFIX=$Julia_PREFIX \
+    -DCMAKE_SPOT_DIR=${prefix} \
+    -DCMAKE_FIND_ROOT_PATH=$prefix \
+    -DJlCxx_DIR=${libdir}/cmake/JlCxx \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    ${macos_extra_flags} \
+    ${windows_extra_flags} \
+    -DCMAKE_BUILD_TYPE=Release \
+    ..
+VERBOSE=ON cmake --build . --config Release --target install -- -j${nproc}
+install_license $WORKSPACE/srcdir/spot_julia/LICENSE.md
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("x86_64", "windows"),
+    Platform("x86_64", "macos")
+]
+platforms = expand_cxxstring_abis(platforms)
+
+# # uncomment when pushing to yggdrasil
+# # These are the platforms we will build for by default, unless further
+# # platforms are passed in on the command line
+# include("../../L/libjulia/common.jl")
+# platforms = libjulia_platforms(julia_version)
+# platforms = filter!(!Sys.iswindows, platforms) # Singular does not support Windows
+# platforms = expand_cxxstring_abis(platforms)
+
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("genltl", :genltl, "bin"),
+    LibraryProduct("libspot", :libspot, "lib"),
+    ExecutableProduct("ltl2tgta", :ltl2tgta, "bin"),
+    ExecutableProduct("ltlsynt", :ltlsynt, "bin"),
+    ExecutableProduct("ltlcross", :ltlcross, "bin"),
+    LibraryProduct("libspotgen", :libspotgen, "lib"),
+    ExecutableProduct("autcross", :autcross, "bin"),
+    ExecutableProduct("genaut", :genaut, "bin"),
+    ExecutableProduct("ltl2tgba", :ltl2tgba, "bin"),
+    ExecutableProduct("randaut", :randaut, "bin"),
+    ExecutableProduct("autfilt", :autfilt, "bin"),
+    ExecutableProduct("ltlfilt", :ltlfilt, "bin"),
+    ExecutableProduct("ltlgrind", :ltlgrind, "bin"),
+    ExecutableProduct("ltldo", :ltldo, "bin"),
+    LibraryProduct("libbddx", :libbddx, "lib"),
+    LibraryProduct("libspotltsmin", :libspotltsmin, "lib"),
+    ExecutableProduct("randltl", :randltl, "bin"),
+    ExecutableProduct("dstar2tgba", :dstar2tgba, "bin"),
+    LibraryProduct("libspot_julia", :libspot_julia, "lib")
```

Comment:
You don't need all the directories here.

Reply:
removed in the latest commit thanks!